README.md

```
@@ -36,6 +44,17 @@ Every adopted probe will generate additional daily credits that you can use to r
 
 Learn more about [Globalping credits](https://globalping.io/credits)
 
+### Automated adoption
+
+For full automation you can auto-adopt all probes under your account without any manual work.
+You simply need to find your adoption token in your [dashboard settings](https://dash.globalping.io/settings) or start a probe adoption process and the full command will be provided for you.
+
+For example (replace with your own unique token)
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)